templates/admin/repo/unadopted.tmpl

```diff
@@ -61,7 +70,16 @@
 												<input type="hidden" name="action" value="delete">
 												<input type="hidden" name="q" value="{{$.Keyword}}">
 												<input type="hidden" name="page" value="{{$.CurrentPage}}">
-												{{template "base/delete_modal_actions" .}}
+												<div class="actions">
+													<button class="ui red basic inverted cancel button">
+														{{svg "octicon-trash" 16 "gt-mr-2"}}
+														{{$.locale.Tr "modal.no"}}
+													</button>
+													<button class="ui green basic inverted ok button">
+														{{svg "octicon-check" 16 "gt-mr-2"}}
+														{{$.locale.Tr "modal.yes"}}
+													</button>
+												</div>
```

Comment:
Why not make another subtemplate with the trash button? 3 repetitions is already worthy of a subtemplate.

Reply:
It just need a rename.

Rename the `delete_modal_actions` to `modal_actions_yes_no`

Reply:
Either is  fine, fix bugs first 😁